lib/services/android-project-service.ts

```diff
@@ -258,6 +258,23 @@ export class AndroidProjectService extends projectServiceBaseLib.PlatformProject
 	public buildProject(projectRoot: string, buildConfig?: IBuildConfig): IFuture<void> {
 		return (() => {
 			if (this.canUseGradle().wait()) {
+				// Validate App_Resources drawable resources accoring to the official docs: https://developer.android.com/guide/topics/resources/providing-resources.html
+				let drawableFilePrefix = "drawable-";
+				let allowedDrawables = ["ldpi", "mdpi", "hdpi", "xhdpi", "xxhdpi", "xxxhdpi", "nodpi", "tvdpi"];
+				let appSourceDirectoryPath = path.join(this.$projectData.projectDir, constants.APP_FOLDER_NAME);
+				let sourceAppResourcesDirectoryPath = path.join(appSourceDirectoryPath, constants.APP_RESOURCES_FOLDER_NAME);
```

Comment:
isn't something missing here  - the path will be `<project dir>/app/App_Resources`, while in fact you need `Android` dir if I'm not mistaken. Later you are using `enumerateFilesInDirectorySync` which will include all files from App_Resources, so you'll evalute any files/directories inside iOS dir as well.

I suggest adding Android platform in the path:
```TypeScript
let sourceAppResourcesDirectoryPath = path.join(appSourceDirectoryPath, constants.APP_RESOURCES_FOLDER_NAME, this.$devicePlatformsConstants.Android);
```